src/main/java/htsjdk/samtools/cram/structure/ContainerIO.java

```diff
@@ -42,17 +40,6 @@ public static Container readContainer(final Version version, final InputStream i
         return container;
     }
 
-    /**
-     * Reads next container from the stream.
-     *
-     * @param inputStream the stream to read from
-     * @return CRAM container or null if no more data
-     * @throws IOException
-     */
-    private static Container readContainer(final int major, final InputStream inputStream) throws IOException {
```

Comment:
Merged with the other method of the same name.  The other was never called, so we can remove its 2 parameters